From: Jon Bernard <jbernard@debian.org>
Date: Sun, 27 Mar 2011 21:25:29 -0400
Subject: Fix cgconfig.conf manpage formatting error
To: libcg-devel@lists.sourceforge.net

Lines that begin with a single quote are interpreted as macros instead
of a literal single quote.
---
 doc/man/cgconfig.conf.5 |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/man/cgconfig.conf.5 b/doc/man/cgconfig.conf.5
index 879c1ba..10b5acc 100644
--- a/doc/man/cgconfig.conf.5
+++ b/doc/man/cgconfig.conf.5
@@ -12,8 +12,8 @@ The file consists of
 .I mount
 and
 .I group
-sections. These sections can be in arbitrary order. Any line starting with
-'#' is considered as a comment line and is ignored.
+sections. These sections can be in arbitrary order. Any line starting with '#'
+is considered as a comment line and is ignored.
 .LP
 .I mount
 section has this form:
-- 
